app/api/data_layers/ChargesLayer.py

```diff
@@ -0,0 +1,104 @@
+from flask_rest_jsonapi.data_layers.base import BaseDataLayer
+
+from app.api.helpers.exceptions import UnprocessableEntity
+from app.api.helpers.ticketing import TicketingManager
+from app.models.order import Order
+
+
+class ChargesLayer(BaseDataLayer):
+    def before_create_relationship(self, json_data, relationship_field, related_id_field, view_kwargs):
```

Comment:
Actually, the base class is AbstractClass, so I guess we have to implement every method of it.